open_close.py

```diff
@@ -0,0 +1,48 @@
+import datetime
+
+vent_status = input('enter the vent status')
+
+
+def open_or_close(vent_status):
+    is_input_valid(vent_status)
+    wanted_status = get_wanted_vent_status()
+    if vent_status == wanted_status:
+        print('you dont need to change the status')
+    else:
+        print('you need to change the status')
+
+
+def is_input_valid(situation):
+    if situation == 'open' or situation == 'close':
+        return
+    else:
+        print('The inserted situation is not accepted')
+        exit()
+
+
+def get_wanted_vent_status():
+    current = datetime.datetime.now().time()
```

Comment:
A thing I think worth knowing: 
The `current` param you created, is from type `datetime.time`. (you can find out the type by `print(type(current))`.

Objects of `datetime.time` types has some cool features. 
One that I think is relevant, is the `.hour`, `.minute` , `.second` attributes.
(Attribute - a field of the object, you can access, by dot and the name for the field after the dot.)
For example:
`this_hour = current.hour`

Each of this 3 attributes return the relevant data (for example - current.hour will return a int number of the   current hour).

That attributes can simply your method logic:

you can replace the:
```python
start = datetime.time(12, 0, 0)
end = datetime.time(16, 0, 0)
time = start <= current <= end
if time == True:
 ```
 
 with the simple option:
 
 ```python
 if current.hour >= 12 and current.hour <= 16:
 ```
 
 Remember - a short code is easier to read :)